process_manager/templates/process_manager/index.html

```diff
@@ -64,6 +64,17 @@
     end
   end
   </script>
+  <script>
+    document.addEventListener('DOMContentLoaded', function() {
+      var timezoneElement = document.getElementById('timezone');
+      var timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+      var offset = new Date().getTimezoneOffset();
+      var offsetHours = Math.abs(offset / 60);
+      var offsetSign = offset <= 0 ? '+' : '-';
+      var offsetString = offsetSign + String(offsetHours).padStart(2, '0')
+      timezoneElement.textContent = timezone + ' (UTC' + offsetString + ')';
```

Comment:
My JS knowledge is non-existent, so just to confirm I get this right, this line here will update the HTML element below with `id="timezone"` giving it the value `TIMEZONE (UTC + OFFSET_HOURS)`, both TIMEZONE and OFFSET calculated here, right?

I'd add `h` to the string. Not sure if standard, but makes it clearer. 
```suggestion
      timezoneElement.textContent = timezone + ' (UTC' + offsetString + 'h)';
```

Also, there are some [fractional timezones](https://www.timeanddate.com/time/time-zones-interesting.html) which I'm not sure if will be handled properly with this.

Reply:
Yeah, exactly it will update the element with that ID. 
I did not take into account the fractional timezones will amend this.

Reply:
Currently, it shows this if it is a fractional timezone
![image](https://github.com/user-attachments/assets/63cf7a17-9d6c-48c6-89ea-7c2ec6bf8ab6)